roles/weave/tasks/main.yml

```diff
@@ -29,21 +23,29 @@
   tags:
     - weave
 
+# Create weave bridge.
 - name: bring up weave bridge
   command: ifup weave
   sudo: yes
+
+- name: upload weave template service
```

Comment:
The script is uploaded here. Dont need any notify as it has:
"start on started docker"
It is only going to be run onces docker restart in the next task